test/process-timeouts/index.1.spec.js

```diff
@@ -0,0 +1,5 @@
+describe('suite 1', () => {
+    it('hoard the process', (done) => {
+        setTimeout(done, 10000);
```

Comment:
mocha-parallel-tests are executed sequentially, so if you need to have some asynchronous test better use 3 seconds timeout.

Reply:
This big timeout is meant to be bad, and time out the process.